app/controllers/api/base_controller.rb

```diff
@@ -21,7 +21,7 @@ class Api::BaseController < ApplicationController
 
   cache_api_action :index
 
-  caches_action :entrypoint, cache_path: ->(c){ {_c: Api.version(api_version).cache_key } }
+  caches_action :entrypoint, cache_path: ->(c) { { _c: Api.version(api_version).cache_key } }
```

Comment:
Unused block argument - `c`. If it's necessary, use `_` or `_c` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.<br>Line is too long. [93/80]